tools/topology/topology2/include/pipelines/cavs/passthrough-wov.conf

```diff
@@ -0,0 +1,144 @@
+#
+# CAVS passthrough capture pipeline
```

Comment:
passthrough?

Reply:
CAVS passthrough wov pipeline? I'm not sure whether this is passthough or not. The data is not processed by gain and directly passed to WoV component. Or ```CAVS dummy WoV pipeline```. If so, I need to rename the filename.